packages/web-twig/src/Resources/components/TextArea/README.md

```diff
@@ -34,29 +34,29 @@ Without lexer:
-| `name`            | `string`                      | `null`  | no       | TextArea name                                              |
-| `placeholder`     | `string`                      | `null`  | no       | TextArea placeholder                                       |
-| `rows`            | `number`                      | `null`  | no       | Number of visible rows                                     |
-| `validationState` | `success`, `warning`, `error` | `null`  | no       | Type of validation state                                   |
-| `value`           | `string`                      | `null`  | no       | TextArea value                                             |
+| Prop name         | Type                                    | Default | Required | Description                                                |
+| ----------------- | --------------------------------------- | ------- | -------- | ---------------------------------------------------------- |
+| `class`           | `string`                                | `null`  | no       | Custom CSS class                                           |
+| `id`              | `string`                                | —       | yes      | TextArea and label identification                          |
+| `isDisabled`      | `bool`                                  | `false` | no       | If true, TextArea is disabled                              |
+| `isFluid`         | `bool`                                  | `false` | no       | If true, the element spans to the full width of its parent |
+| `isLabelHidden`   | `bool`                                  | `false` | no       | If true, label is hidden                                   |
+| `isRequired`      | `bool`                                  | `false` | no       | If true, TextArea is required                              |
+| `label`           | `string`                                | —       | yes      | Label text                                                 |
+| `maxLength`       | `number`                                | `null`  | no       | Maximum number of characters                               |
+| `message`         | `string`                                | `null`  | no       | Validation or help message                                 |
+| `name`            | `string`                                | `null`  | no       | TextArea name                                              |
+| `placeholder`     | `string`                                | `null`  | no       | TextArea placeholder                                       |
+| `rows`            | `number`                                | `null`  | no       | Number of visible rows                                     |
+| `validationState` | `success`, `warning`, `danger`, `error` | `null`  | no       | Type of validation state                                   |
```

Comment:
link to docs/dictionaries and deprecation info as well please

Reply:
done